pkg/wpa/passphrase/passphrase_test.go

```diff
@@ -0,0 +1,101 @@
+package passphrase
+
+import (
+	"bytes"
+	"testing"
+)
+
+type RunTestCase struct {
+	name      string
+	essid     string
+	pass      string
+	out       []byte
+	errExists bool
+}
+
+var (
+	essidStub     = "stub"
+	shortPass     = "aaaaaaa"                                                          // 7 chars
+	longPass      = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" // 64 chars
+	validPass     = "aaaaaaaaaaaaaaaa"                                                 // 16 chars
+	correctOutput = []byte(
+		`network={
+	ssid="stub"
+	#psk="aaaaaaaaaaaaaaaa"
+	psk=e270ba95a72c6d922e902f65dfa23315f7ba43b69debc75167254acd778f2fe9
+}
+`) // indentation matters
+
+	runTestCases = []RunTestCase{
+		{
+			name:      "No essid",
+			essid:     "",
+			pass:      validPass,
+			out:       nil,
+			errExists: true,
+		},
+		{
+			name:      "pass length is less than 8 chars",
+			essid:     essidStub,
+			pass:      shortPass,
+			out:       nil,
+			errExists: true,
+		},
+		{
+			name:      "pass length is more than 63 chars",
+			essid:     essidStub,
+			pass:      longPass,
+			out:       nil,
+			errExists: true,
+		},
+		{
+			name:      "Correct Input",
+			essid:     essidStub,
+			pass:      validPass,
+			out:       correctOutput,
+			errExists: false,
+		},
+	}
+)
+
+// printError provides a standard way to print out error message when a test case fails
+func printError(t *testing.T, funcCallStmt string, expOut string, expErrExists bool, actualOut string, actualErr error) {
```

Comment:
just put it inline since you only call it once.

Reply:
Fixed